src/openmethane_prior/omInputs.py

```diff
@@ -74,7 +74,17 @@
 if os.path.exists(domainPath):
     with xr.open_dataset(domainPath) as dss:
         domainXr = dss.load()
-        domainProj = pyproj.Proj(proj='lcc', lat_1=domainXr.TRUELAT1, lat_2=domainXr.TRUELAT2, lat_0=domainXr.MOAD_CEN_LAT, lon_0=domainXr.STAND_LON, a=6370000, b=6370000)
+        domainProj = pyproj.Proj(
+            proj='lcc',
+            lat_1=domainXr.TRUELAT1,
+            lat_2=domainXr.TRUELAT2,
+            lat_0=domainXr.MOAD_CEN_LAT,
+            lon_0=domainXr.STAND_LON,
+            x_0=domainXr.XORIG,
+            y_0=domainXr.YORIG,
```

Comment:
Adding values for `x_0` and `y_0` is the actual change here. I need to verify with @prayner that this is correct and won't have unintended side effects elsewhere.

The values are `XORIG: -2270000` and `YORIG: -2165629.25` which appear to be offsets in meters from the projection centre lat/long to the southwestern corner of the grid.

Reply:
If the projection is wrong, then I wonder if this impacts #17  since this used to reproject the night time lights which I think are used for distributing industrial emissions

Reply:
My inclination is to comment this line out and possibly leave a reference to #17 or even a new issue. The purpose of this PR is to get what's in the domain file into JSON, and I think that's been achieved.

If we end up making this change in a follow-up PR, the JSON export code will continue to work as-is.

Reply:
I've raised an issue to follow up on this: https://github.com/openmethane/openmethane-prior/issues/24.

For now, I've commented the change out in [eb84611](https://github.com/openmethane/openmethane-prior/pull/11/commits/eb84611f2edeaf0f40f55f2c92be0946b5bdf2fe) but I do think this is moving in the right direction and you're probably right about it being the cause of #17 because I can see the projection is converted to a CRS in that file, which won't work without the coordinate system origin set.